src/styles/index.scss

```
@@ -1,3 +1,124 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap';
+
+$main-accent: #060B35;
+$secondary: #616070;
+$blue-accent: #00ACDC;
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  margin: 0 auto;
+
+  width: 200px;
+  height: 408px;
+
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__imac {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__bottom {
+    display: flex;
+    flex-direction: column;
+
+    margin-top: 40px;
+  }
+
+  &__title {
+    color: $main-accent;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+
+  &__subtitle {
+    font-size: 10px;
+    color: $secondary;
+    font-weight: 400;
+    line-height: 14px;
+    margin-bottom: 16px;
+  }
+
+  &__cost {
+    display: flex;
+    justify-content: space-between;
+
+    margin-bottom: 16px;
+    margin-top: 24px;
+  }
+
+  &__price {
+    color: $secondary;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+  }
+
+  &__sum {
+    color: $main-accent;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+  }
+
+  &__buy {
+    display: block;
+    box-sizing: border-box;
+    background-color: $blue-accent;
+    border-radius: 5px;
+    padding: 11.5px 70px;
+    text-align: center;
+
+    color: white;
+    font-weight: 700;
+    font-size: 14px;
+    text-transform: uppercase;
+    text-decoration: none;
+
+    &:hover {
+      background-color: white;
+      color: $blue-accent;
+      border: 1px solid $blue-accent;
+    }
+  }
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  &__container {
+    display: flex;
+  }
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+
+    &:nth-child(-n+4) {
+      background-image: url(../images/star-active.svg);
+    }
+  }
+
+  &__reviews {
+    color: $main-accent;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
```

```suggestion
  &__reviews {
    color: $main-accent;
    font-size: 10px;
    font-weight: 400;
    line-height: 14px;
  }
```

That`s should be a part of card block